client/components/fields/editor/AssociatedEvent.tsx

```diff
@@ -58,10 +71,22 @@ export class EditorFieldAssociatedEventComponent extends React.PureComponent<IAs
 
         return (
             <div>
-                <label className="InputArray__label side-panel__heading side-panel__heading--big">
-                    {gettext('Related Events')}
-                </label>
-
+                <Spacer h gap="4" justifyContent="space-between" noWrap>
+                    <label className="InputArray__label side-panel__heading side-panel__heading--big">
+                        {gettext('Related Events')}
+                    </label>
+                    {disabled !== true && this.props.item._id.includes('temp') === false && (
```

Comment:
> `this.props.item._id.includes('temp')`

This is too unreliable. For example this would match:

`c6f4ddc7-4fe2-436d-9b88-temp3bf98478`

We're probably using UUID v4 and only a-f letters are possible, but still, not a good practice. Use `isTemporaryId` helper that we have already.